packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -16172,8 +16172,134 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /events/attributes:
+    get:
+      tags:
+        - events
+      summary: Get Attribute events
+      description: |
+        Retrieve list of Attribute events that occurred after the specified event ID.
+        Results are ordered in ascending order (oldest events first).
+      operationId: getAttributeM2MEvents
+      parameters:
+        - in: query
+          name: lastEventId
+          description: |
+            ID of the last event already received.
+            Only events that occurred after this ID will be returned.
+            If not provided, the oldest events up to the specified limit will be returned.
+          schema:
+            type: string
+            format: uuid
+        - in: query
+          name: limit
+          description: Maximum number of results to return
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
```

For M2M events the limit is 500
```suggestion
            maximum: 500
```